rule/rule_test.go

```diff
@@ -362,7 +362,8 @@ func TestAddFilter(t *testing.T) {
 		}
 		assert.EqualValues(t, arg3Field, rule.fields[0])
 		assert.EqualValues(t, equalOperator, rule.fieldFlags[0])
-		assert.EqualValues(t, math.MaxUint32, rule.values[0])
```

Comment:
I'm curious as to what issue occurred here.

Reply:
Turns out `math.MaxUint32` is defined as an int. I've submitted a nicer fix.